doc/content/getting-started/user-management/user/_index.md

```diff
@@ -1,12 +1,63 @@
 ---
 title: "User Management"
 description: ""
+aliases: [/getting-started/user-management/cli/user, /getting-started/user-management/console/user]
 ---
 
-This section contains instructions for managing users with the CLI.
+This section contains instructions for managing users.
 
 <!--more-->
 
+{{< tabs/container "Console" "CLI" >}}
+
+{{< tabs/tab "Console" >}}
+
+## Managing Users in the Console
+
+User management can be found in the user dropdown in the top right corner of the Console.
+
+{{< figure src="dropdown.png" alt="User Dropdown" >}}
+
+## Listing Users
+
+The list of users is shown immediately after going to **User Management** in the user dropdown.
```

Comment:
```suggestion
The list of users is shown immediately after going to **User management** in the user dropdown.
```

Reply:
There are likely more occurrences of wrong casing for console button references. We should successively fix those.